filter/index.js

```diff
@@ -1,19 +1,31 @@
+// function onlyEven(array) {
+//   const evens = [];
+//   for (let i = 0; i < array.length; i++) {
+//     if (array[i] % 2 === 0) {
+//       evens.push(array[i]);
+//     }
+//   }
+//   return evens;
+// }
+
+// refactor with filter!! OMG!
 function onlyEven(array) {
-  const evens = [];
-  for (let i = 0; i < array.length; i++) {
-    if (array[i] % 2 === 0) {
-      evens.push(array[i]);
-    }
-  }
-  return evens;
+  var evens = array.filter(function(even){
+    return (even % 2 === 0)
+  })
```

Comment:
Nice! we'd like you to try using es6 change your var to `const` and replace the anonymous function with a fat arrow
```js
return array.filter(n => n % 2 === 0);
```